resources/task.rb

```diff
@@ -0,0 +1,149 @@
+property :task_name, String, desired_state: false, identity: true, name_attribute: true
+property :task_source, String, desired_state: false, identity: true, default: ''.freeze
+property :task_crontab, String, desired_state: false, identity: true, default: '0 1 * * * ?'.freeze
+property :api_endpoint, String, desired_state: false, identity: true,
+                                default: lazy { node['nexus3']['api']['endpoint'] }
+property :api_username, String, desired_state: false, identity: true,
+                                default: lazy { node['nexus3']['api']['username'] }
+property :api_password, String, desired_state: false, identity: true, sensitive: true,
+                                default: lazy { node['nexus3']['api']['password'] }
+
+load_current_value do |desired|
+  api_endpoint desired.api_endpoint
+  api_username desired.api_username
+  api_password desired.api_password
+
+  def apiclient
+    @apiclient ||= ::Nexus3::Api.new(api_endpoint, api_username, api_password)
+  end
+
+  begin
+    res = apiclient.run_script('get_task', desired.task_name)
+    current_value_does_not_exist! if res == 'null'
+    config = JSON.parse(res)
+    ::Chef::Log.warn "Config is: #{config}"
+    task_name config['.name']
```

Comment:
This is an identity property, so there is no need to set it. And could you explain me why there is a *.* before `name` ?

Reply:
removed.